Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -152,6 +168,8 @@ FieldValue Serializer::DecodeFieldValue(
       return FieldValue::NullValue();
     case FieldValue::Type::Boolean:
       return FieldValue::BooleanValue(value_proto.value.boolean_value);
+    case FieldValue::Type::Long:
```

Comment:
The mismatch is obvious here too: could you rename `FieldValue::Type::Long` to match above?